ui_framework/components/tabs/tab.test.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { render, shallow } from 'enzyme';
+import { requiredProps } from '../../test/required_props';
+import sinon from 'sinon';
+
+import {
+  KuiTab,
+} from './tab';
+
+describe('KuiTab', () => {
+  test('renders', () => {
+    const component = <KuiTab { ...requiredProps }>children</KuiTab>;
+    expect(render(component)).toMatchSnapshot();
+  });
+
+  test('renders isSelected', () => {
+    const component = <KuiTab isSelected { ...requiredProps }>children</KuiTab>;
+    expect(render(component)).toMatchSnapshot();
+  });
+
+  describe('Props', () => {
+    describe('onClick', () => {
+      test(`isn't called upon instantiation`, () => {
```

Comment:
Is that something that needs to be tested? We do not test for all other things that do not happen on instantiation either.